README.md

```diff
@@ -63,6 +63,30 @@ additional information on stdout.
 
 This requires SDL.
 
+Build-time requirements
+-----------------------
+
+### make
+
+While we are trying to keep our makefiles portable,
+it might be incompatible with some versions of make.
+
+#### GNU make 4.x
+
+It works. We recommend to use it.
+
+#### BSD make
+
+It also works.
+You might need to specify the `-r` make option because some of
+the default macros like CFLAGS from sys.mk can cause unintended effects.
+
+#### GNU make prior to 4.0
+
+It doesn't work because it doesn't support `!=`.
+
+*Note*: macOS's default version of make is GNU make 3.81 as of writing this.
```

Comment:
I think it would be easier for readers if we had just one GNU make section and elaborate which version(s) work and don't within. At is stand, I find confusing that the BSD make section is "sandwiched" between two GNU make sections.

Reply:
it makes sense. done.